Templates/HttpPUT(CRUD)-CSharp/run.csx

```diff
@@ -0,0 +1,26 @@
+﻿#r "Microsoft.WindowsAzure.Storage"
+
+using System.Net;
+using Microsoft.WindowsAzure.Storage.Table;
+
+public static HttpResponseMessage Run(Person person, CloudTable outTable, TraceWriter log)
+{
+    if (string.IsNullOrEmpty(person.Name))
+    {
+        return new HttpResponseMessage(HttpStatusCode.BadRequest)
+        {
+            Content = new StringContent("Person entity not present in HttpRequest body")
```

Comment:
This error message doesn't really seem correct. Might be a person w/o a Name property. Perhaps this is just a wording issue. If your intent is to do validation on the incoming object, then perhaps "A non-empty Name must be specified."